### PR TITLE
Capabilities feature

### PR DESCRIPTION
**Pull request by elavarasidc**
## Purpose
Introducing capabilities feature to get the capabilities of a local participant and listen to capabilities update.

## Does this introduce a breaking change?

```
[ ] Yes
[x] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?

```
[ ] Bugfix
[x] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Documentation content changes
[ ] Other... Please describe:
```

## How to Test
*  Get the code

```
git clone [repo-address]
cd [repo-name]
git checkout [branch-name]
npm install
```

* Test the code

```
Steps	Action	Expected result	Attachments
1.	ACS User joins an ACS Room
2.	ACS user registers for capabilities feature	
3.	ACS user successfully get's the users capabilities in the meeting setup
4.	ACS user registers listeners to listen to capability updates
5.	ACS user receives capability updates event
```

## Other Information
Capabilities feature - Only role based capabilities has been implemented for ACS Rooms

### CODE REVIEW

File: Project/src/MakeCall/CallCard.js [hunk 1]
```diff
@@ -313,6 +313,14 @@ export default class CallCard extends React.Component {
                 this.setState({ dominantRemoteParticipant: utils.getRemoteParticipantObjFromIdentifier(dominantSpeakerIdentifier) })
             }
             this.call.feature(Features.DominantSpeakers).on('dominantSpeakersChanged', dominantSpeakersChangedHandler);
+
+            const capabilitiesFeature =  this.call.feature(Features.Capabilities);
+            const capabilities =  this.call.feature(Features.Capabilities).capabilities;
+            console.log(`Local Participant capabilities are=${JSON.stringify(capabilities)}`);
+            capabilitiesFeature.on('capabilitiesChanged', () => {
+                const updatedCapabilities  = capabilitiesFeature.capabilities;
+                console.log(`EVENT, Local participant capabilities has changed`);
```
Comment (fizampou) [inline Project/src/MakeCall/CallCard.js:322]: remove console logs please

File: Project/src/MakeCall/MakeCall.js [hunk 2]
```diff
@@ -230,6 +231,21 @@ export default class MakeCall extends React.Component {
...
 
+    joinRooms = async (withVideo) => {
+        try {
+            if(this.roomsId.value) {
+                const callOptions = await this.getCallOptions(withVideo);
+                this.callAgent.join({ meetingId: this.roomsId.value }, callOptions);
+            } else {
+                throw new Error('Please enter Rooms ID to join a Rooms call');
+            }
+
```
Comment (fizampou) [inline Project/src/MakeCall/MakeCall.js:242]: you need to render that error not throw it

File: Project/src/MakeCall/CallCard.js (hunk 1 above)
Comment (elavarasidc) [reply Project/src/MakeCall/CallCard.js]: Fixed

File: Project/src/MakeCall/MakeCall.js (hunk 2 above)
Comment (elavarasidc) [reply Project/src/MakeCall/MakeCall.js]: Fixed.

**Review by fizampou — APPROVED** [review]

**Pull request merged**